<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout100">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>966</width>
        <height>769</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout100">
           <item>
            <widget class="QGroupBox" name="graf1GroupBox">
             <property name="title">
              <string>IME KOMPONENTE</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="graf1Layout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="graf2GroupBox">
             <property name="title">
              <string>IME KOMPONENTE</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="graf2Layout"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="rezultatiGroupBox">
             <property name="title">
              <string>IME KOMPONENTE</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="rezultatLayout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="slopeGroupBox">
               <property name="title">
                <string>IME KOMPONENTE</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="slopeLayout"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="kriterijGroupBox">
             <property name="title">
              <string>IME KOMPONENTE</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="kriterijLayout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
